airbyte-integrations/connectors/source-s3/source_s3/v4/source.py

```diff
@@ -21,7 +21,7 @@
 
 
 class SourceS3(FileBasedSource):
-    _concurrency_level = DEFAULT_CONCURRENCY
+    _concurrency_level = None
```

Comment:
Note: i'll remove this before merging, but I turned this off to test that this worked on the platform for synchronous file-based sources